projects/ion/src/lib/input-counter/input-counter.component.html

```diff
@@ -9,14 +10,18 @@
 
   <input
     type="text"
-    oninput="this.value = this.value.replace(/[^0-9]/g, '').replace(/(\..*)\./g, '$1');"
-    [(ngModel)]="count"
     data-testid="input-count"
-    (ngModelChange)="changeCount($event)"
+    maxlength="9"
```

Comment:
I'm also not sure about this, have you aligned this with the design team?

Reply:
The user can currently enter a very large number, which could potentially break the code. If this functionality seems unnecessary, we could either remove it or increase the digit limit

Reply:
The functionality its fine to me, I was just wondering if this value is the better option, thats why I suggested to ask the design team's opinion. Maybe even transform into a input so the dev will decide wich value he wants as maxLength.

Reply:
Turning it into an `Input` looks good, but it still needs a default value. I’ll check with design to define it.